test/integration_test_helper.rb

```diff
@@ -20,18 +20,40 @@
 end
 
 Selenium::WebDriver::Chrome::Service.driver_path = ENV['TESTDRIVER_PATH'] || Foreman::Util.which('chromedriver', Rails.root.join('node_modules', '.bin'))
+
+options = Selenium::WebDriver::Chrome::Options.new
+options.args << '--disable-gpu'
+options.args << '--no-sandbox'
+options.args << '--window-size=1024,768'
```

Comment:
Do we support this resolution, shouldn't it be 1920,1080?

Reply:
That part just moved code.

So while "yeah, probably", it's not a change in this PR :)

Reply:
Also I think some tests relay on the screen to be that size.

Reply:
In https://github.com/theforeman/foreman/pull/9962 I'm dropping it. That currently fails, but I don't know if it's because of me or something else.

Reply:
@ekohl See my comment https://github.com/theforeman/foreman/pull/9962/files#r1432594792 to possible cause of error

Reply:
@ekohl I based my PR on your changes from https://github.com/theforeman/foreman/pull/9962 and now the integration tests are failing.

I would assume this is because of the missing screen size option?

I would propose to first fix and merge your PR and then we can continue with this one.

Reply:
Since there wasn't any progress on #9962 I incorporate all the changes here.

There is still one integration test failing. This might be due to the screen size option?

About the removed `--headless` option: The only way I found to pass chrome options to capybara is to register the driver. When doing this the options set in capybara https://github.com/teamcapybara/capybara/blob/3.39.2/lib/capybara/registrations/drivers.rb#L31 are overwritten. So if we want to set additional options like screen size we have to set `--headless` for `:selenium_chrome_headless`.  

Let me know in which direction we should move. Any help is greatly appreciated.

Reply:
The failing test is probably due to screen size. it tests if there is a column named "locations" in smart proxies page, but its hidden when the screen size is less than `991px`

Reply:
Thank you, tests are green now. So I guess we need to keep the screen size option then.